askup/templates/askup/question_answer_modal.html

```diff
@@ -9,7 +9,7 @@ <h4 class="correct" id="your-answer-read-only"></h4>
                 <br>
             </div>
             <div class="hide-on-answered">
-                <form id="answer-form" method="post" action="{% url 'askup:question_answer' 1 %}">
+                <form id="answer-form" method="post" action="{% url 'askup:question_answer' 1 1 %}">
```

Comment:
We've decided that using args name will be more clear to understand what params are sent.

Reply:
this part of code is already deleted in the next PR. made this changes for the sake of compatibility.

Reply:
Ok, got it.